samples/cellular/nrf_cloud_multi_service/overlay-nrf7002ek-wifi-scan-only.conf

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-# Overlay to use nRF7002 EK on top of nrf9160 DK for Wi-Fi scanning
+# Overlay to use nRF7002 EK on top of nrf91 Series DK for Wi-Fi scanning
```

Comment:
Have you tested this overlay with nrf9161?

Reply:
No, I don't have a nRF7002EK. My assumption was that we will have the same support for nRF9161 DK as with nRF9160 DK.